<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelSteps">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Steps :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDispalySteps">
     <property name="text">
      <string>DisSteps</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLaps">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Laps :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDispalyLaps">
     <property name="text">
      <string>DisLaps</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
